spec/spec_helper.rb

```diff
@@ -5,6 +5,7 @@
 require 'knapsack'
 Dir["./spec/models/concerns/*.rb"].each { |f| require f }
 Dir["./spec/support/**/*.rb"].sort.each { |f| require f }
+Dir["./spec/shared/**/*.rb"].sort.each { |f| require f }
```

Comment:
Seems like we have 4 shared_examples for concerns, but they are across 3 different subfolder 🙈 . I'll correct this afterwards don't worry :). IMHO `/spec/models/concerns/` should be the path for a concern shared_example